deploy/handler/operator.yaml

```diff
@@ -28,6 +28,13 @@ spec:
       nodeSelector: {{ toYaml .InfraNodeSelector | nindent 8 }}
       tolerations: {{ toYaml .InfraTolerations | nindent 8 }}
       affinity: {{ toYaml .WebhookAffinity | nindent 8 }}
+      topologySpreadConstraints:
+      - maxSkew: 1
+        topologyKey: kubernetes.io/hostname
```

Comment:
Shouldn't the topologyKey be `node-role.kubernetes.io/control-plane`?
Otherwise, I don't understand what is the purpose of adding second master node

Reply:
Ah, nope, now I get the `topologySpreadConstraints` behaviour.
Zoning by hostname makes sense, since we don't really know apriori other labels that would distinguish nodes.

So, I don't understand why node2 is labeled as master then.

Reply:
Topology key is the value that mark pods uniquedness, I have mark the node02 as master to be able to do e2e test checking that webhook is deployed at both.